site/src/pages/release-notes/1.14.1.mdx

```diff
@@ -0,0 +1,46 @@
+---
+date: 2022-2-16
+---
+
+## 🌟 New features
+
+- N/A
+
+## 📈 Improvements
+
+- Users have more transparency on why they encounter an `EmptyEndpointGroupException` #4084
+
+## 🛠️ Bug fixes
+
+- `DnsEndpointGroup` now correctly respects a trailer dot of a hostname when resolving DNS records. #4088
+- You can disable the ignored multiple routes warning log by specifying `-Dcom.linecorp.armeria.reportIgnoredMultipleRoutes=false`. #4086
+- `GrpcClientBuilder` now correctly normalizes a path prefix. #4085
+- `GrpcClientBuilder.path()` has been deprecated in favor of `GrpcClientBuilder.pathPrefix()`. #4085
```

Comment:
Is this a bug or deprecation?

Reply:
I think #4085 did both (bug fix + deprecation), so I added the PR in each section.
Previously, the message was duplicated though, so I fixed it 👍